### PR TITLE
Add "Clash" Extension

### PR DESCRIPTION
**Pull request by ciiiii**
## Description

Monitor your [Clash](https://github.com/Dreamacro/clash) with Raycast.
Including Proxies, Connections, Rules and Logs data, and support backends switch.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/28784251/147405498-8dc723fe-e603-4d4f-a6fd-c6fdca329895.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/clash/package.json
Comment (pernielsentikaer) [inline extensions/clash/package.json:12]:
```suggestion
      "title": "Dashboard",
      "subtitle": "Clash",
```

File: extensions/clash/package.json
Comment (pernielsentikaer) [inline extensions/clash/package.json:4]:
```suggestion
  "title": "Clash",
```

File: extensions/clash/package.json
Comment (pernielsentikaer) [inline extensions/clash/package.json:13]:
```suggestion
      "description": "List Clash dashboard items",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ciiiii 👋 

I left some suggestions and feedback for you 😄 

- If Clash is not installed: Could you instead then just show empty list [] (That shows No Results) and use the toast to show that Clash is not installed. It's a bit confusing right now that all items are visible but nothing works without Clash.

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by ciiiii** [comment]
@pernielsentikaer, Thank for your reply. 😄 
I don't think we should install clash in this extension, as following reasons:
- There are many version of clash, people may choose different versions, such as Clash、ClashX、ClashPro、Clash for windows for mac....
- It always have network problems to install clash directly in some countries and areas.

Maybe I should add a more friendly toast if no backends config, How do you think about this?

**Comment by pernielsentikaer** [comment]
> I don't think we should install clash in this extension, as following reasons:

I don't want to install Clash in the extension - I'm just talking about showing “No Results” if it's not installed (and showing the toast) - Instead of going in an item where the toast shows something wrong, it should be possible to move the check?

E4XArYifocJe9OiU

**Comment by ciiiii** [comment]
@pernielsentikaer 😅Sorry for misunderstanding your words, I misrecognize "instead" for "install", your suggestion is good, I will improve that.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ciiiii 👋 

It looks very nice, good job 💪 

I was actually thinking about if it is possible to move the “No Results” to the main list like the screenshot instead, so it's not possible to do anything before setup?

zy2WGJiXlsL5lrqn

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by ciiiii** [comment]
> Hi @ciiiii 👋
> 
> It looks very nice, good job 💪
> 
> I was actually thinking about if it is possible to move the “No Results” to the main list like the screenshot instead, so it's not possible to do anything before setup?
> 
> zy2WGJiXlsL5lrqn
> 
> Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

I think the main list shouldn't change, otherwise users may be confused, and "Backends" is always required. Will it be better to add a "no backend" toast in the main list?

**Comment by pernielsentikaer** [comment]
That makes fine sense @ciiiii – a toast in main list if possible would do it 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by ciiiii** [comment]
@pernielsentikaer Error toast in main list is added, wait for your review. 💪

**Review by pernielsentikaer — APPROVED** [review]
Hi @ciiiii 👋 

Thanks for the correction, I like it's much better now 💪 

I'm going to accept it and then will my colleague do a final review before it's finally merged.

**Pull request merged**
